### ISSUE TITLE
Controller - Access to core utils (scrubbed)

### ISSUE THREAD

**Issue by adrian-moisa** [visual-space/visual-editor#6]
There are dozens of utils that could be useful for various tasks around the visual editor. For example one particular useful method is computing the position of the text relative to the window. This is highly useful for placing overlays at a precise position relative to the text. 

Join on [discord](https://discord.gg/XpGygmXde4) to get advice and help or follow us on [YouTube Visual Coding](https://www.youtube.com/channel/UC2-5lfNbbErIds0Iuai8yfA) to learn more about the architecture of Visual Editor and other Flutter apps.

**Comment by adrian-moisa** [comment]
After working more with the editor I realised that this idea is not so great. A lot of the tasks where I needed these utils are better executed as internal lib code rather than public api. It would create too much complexity to expose highlights, markers in the public. In other tickets we approached this task differently. Therefore this idea no longer has merit. I mean there's still some utility to it, but it's not longer required in our MVP plans. Maybe in the future we will expose these features if there's demand. So far we keep it as it is.
